README.md

```diff
@@ -92,10 +92,11 @@ The following instructions are referenced from the [documentation](https://dart.
 
 ### Options
 
-| Option       | Description                                                                                                          | Default |
-| ------------ | -------------------------------------------------------------------------------------------------------------------- | ------- |
-| `cli`        | The `dart` or `flutter` CLI to use to publish the package to the registry.                                           | `dart`  |
-| `publishPub` | Whether to publish the package to the registry. If set to `false`, the `pubspec.yaml` version will still be updated. | `true`  |
+| Option           | Description                                                                                                                                                               | Default |
+|------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------|
+| `cli`            | The `dart` or `flutter` CLI to use to publish the package to the registry.                                                                                                | `dart`  |
+| `publishPub`     | Whether to publish the package to the registry. If set to `false`, the `pubspec.yaml` version will still be updated.                                                      | `true`  |
+| `useVersionCode` | Whether to write version code for every newly bumped version in `pubspec.yaml` ([Learn more](https://developer.android.com/studio/publish/versioning#versioningsettings)) | `false` |
```

Comment:
Can you:
1. Name this option `updateBuildNumber`
2. Reference it as `build number` instead of `version code` in all the places as this is how Flutter references.
3. Update the link to https://docs.flutter.dev/deployment/android#updating-the-apps-version-number